### PR TITLE
[Admin] Crud for admin user

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | https://github.com/Sylius/Sylius/pull/5539
| License         | MIT

### CODE REVIEW

File: features/user/managing_administrators/adding_administrator.feature [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @ui
+    Scenario: Adding a new administrator and log in with its credentials
+        Given I want to create a new administrator
+        When I specify its email as "l.skywalker@gmail.com"
+        And I specify its name as "Luke"
+        And I specify its password as "lightsaber"
+        And I enable it
+        And I add it
+        And I log out
```
Comment (lchrusciel) [inline features/user/managing_administrators/adding_administrator.feature]: Can we merge last three steps to something like:
`Then I should be able to log in as "l.skywalker@gmail.com" authenticated by "lightsaber" password`

File: src/Sylius/Bundle/AdminBundle/Resources/views/Administrator/Field/confirmed.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% if data.enabled %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Administrator/Field/confirmed.html.twig]: Why don't we extract it to Ui component? This one is pretty generic.

File: src/Sylius/Bundle/CoreBundle/Resources/translations/validators.en.yml [hunk 3]
```diff
@@ -29,3 +29,9 @@ sylius:
     product_translation:
         short_description:
             max: Short description must not be longer then {{ limit }} characters.
+    user:
+        email:
+            unique: This email is already used.
+        username:
+            unique: This username is already used.
+            not_blank: Please enter your name.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Resources/translations/validators.en.yml]: Missing blank line

File: src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml [hunk 4]
```diff
@@ -44,10 +44,12 @@ sylius:
         administration: Administration
         administration_dashboard: 'Administration dashboard'
         administration_panel_login: Administration panel login
+        admin_users: Administrators
```
Comment (NeverResponse) [inline src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml]: Shouldn't you be consistent it naming the keys? Here you have `admin_users` translated to `Administrators` which already smells bad, but moreover [here](http://www.huffingtonpost.com/2014/10/20/spotify-family-plan_n_6014606.html), you actually use administrators in the key, so what's up?

File: src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml [hunk 5]
```diff
@@ -429,6 +432,7 @@ sylius:
         n_a: N/A
         name: Name
         new: New
+        new_admin_user: 'New adminstrator'
```
Comment (NeverResponse) [inline src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml]: adminstrator => admin**i**stator

File: src/Sylius/Behat/Resources/config/services/pages/admin/administrator.xml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+            <argument type="string">admin_user</argument>
+        </service>
+        <service id="sylius.behat.page.admin.administrator.index" class="Sylius\Behat\Page\Admin\Crud\IndexPage" parent="sylius.behat.page.admin.crud.index" scope="scenario" public="false">
+            <argument type="string">admin_user</argument>
+        </service>
+        <service id="sylius.behat.page.admin.administrator.update" class="Sylius\Behat\Page\Admin\Administrator\UpdatePage" parent="sylius.behat.page.admin.crud.update" scope="scenario" public="false">
+            <argument type="string">admin_user</argument>
+        </service>
+    </services>
+</container>
```
Comment (NeverResponse) [inline src/Sylius/Behat/Resources/config/services/pages/admin/administrator.xml]: Missing a blank line

**Comment by pjedrzejewski** [comment]
@Arminek I think we should stick to the convention and use "AdminUser" as the entity name, not "Administrators". So ``managing_admin_users``, ``@SyliusAdmin/AdminUser/x.html.twig``, etc.

**Comment by pjedrzejewski** [comment]
Thanks Arek! :)

**Pull request merged**